.gitignore

```diff
@@ -44,4 +44,6 @@ Thumbs.db
 .nx
 
 # env
-.env*
\ No newline at end of file
+.env*
+/packages/sample-app/chromadb
```

Comment:
This is not a good thing to put in a gitignore. It's better to put the chromadb in some known folder with a distinct name like ".temp-chroma"

Reply:
We no longer required this. And we are gonna directly ignore `chroma.sqllite3`